code/Weapons/Gadget/Components/AnvilPhysicsGadgetComponent.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace Grubs;
+
+[Prefab]
+public partial class AnvilPhysicsGadgetComponent : GadgetComponent
+{
+	[Prefab, Net]
+	public bool LockXAxis { get; set; } = true;
+
+	[Prefab, Net]
+	public bool AffectedByWind { get; set; } = false;
+
+	[Prefab, ResourceType( "sound" )]
+	public string CollisionSound { get; set; }
+
+	[Prefab, Net, Predicted]
+	public int Bounces { get; set; } = 0;
```

Comment:
Does changing a prefab value not mess it up for subsequent uses? Typically what we would do is have a prefab variable called `MaxBounces`, then a private variable `_bounces` which would track the remaining bounces.

Reply:
Just did some testing to confirm, and it appears that Prefab fields are populated with their corresponding prefab values each time the object is instantiated. I can still swap this over if we'd like since this is pretty standard.

Reply:
I think whenever we can we should swap to a backing field (since we do that everywhere else). `MaxBounces` and maybe `_bouncesRemaining`.